kuma/static/styles/components/compat-tables/bc-icons.scss

```diff
@@ -4,17 +4,26 @@
 .bc-browsers {
     .bc-head-txt-label {
         display: inline-block;
-        margin: 0 0 0 50px;
+        bottom: 0;
+        margin: 0 0 20px 10px;
```

Comment:
The asymmetry of the margins surprises and confuses me. Assuming you want equal left and right margins, and given the rotation, I'd have thought that you'd have equal top and bottom margins declared here...

Not saying it is wrong; just mentioning the parts that are hard to understand.